include/deal.II/simplex/grid_generator.h

```diff
@@ -27,221 +27,217 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-namespace Simplex
+namespace GridGenerator
```

Comment:
should there be a ``#ifdef DEAL_II_WITH_SIMPLEX_SUPPORT`` here?

Reply:
Do you like this solution?